doc/source/skips/1-values.rst

```diff
@@ -0,0 +1,168 @@
+and the future of the library would be referenced against this document. (See
+:ref:`governance`.)
+
+In July 2018, I published a blog post that broadly outlined what I would want
+from a roadmap for scikit-image [1]_, but requested comments from the community
+before it would be finalized. I consider that we have collected comments for
+long enough and can move forward with formal adoption. Most comments were
+positive, so below I'll just summarize the “negative” comments under “rejected
+ideas”.
+
+Detailed description
+--------------------
+
+(Or: What problem does this proposal solve?)
+
+Over the past few years, scikit-image has been slightly “adrift”, with new and
+old contributors coming in, adding what small bits they need at the time, and
+disappearing again for a while. This is *fine* and we want to encourage more of
+it, but it also lacks direction. Additionally, without a concerted roadmap to
+concentrate effort, many of these contributions just fall by the wayside, as it
+is difficult for new contributors to live up to our stringent (and largely
+unwritten) standards of code.
+
+Implementation
+--------------
+
+Our mission
+***********
+
+scikit-image aims to be the essential toolkit for scientific image analysis in
+Python. We accomplish this by:
+
+- being **easy to use and install**. We are careful in taking on new
+  dependencies, and sometimes cull existing ones, or make them optional.
+- providing a **consistent API**. Conceptually identical arguments have the
+  same name and position in a function signature.
+- **ensuring accuracy**. Test coverage is close to 100% and code is reviewed by
+  at least two core developers before being included in the library.
+- **caring about users’ data**. We have a functional [2]_ API and don't modify
+  input arrays unless explicitly directed to do so.
+- promoting **education** (including self-education) **in image processing**,
+  with extensive pedagogical documentation.
+
+Our values
+**********
+
+- We are inclusive. We continue to welcome and mentor newcomers who are
+  making their first contribution.
+- We are community-driven. Decisions about the API and features are driven by
+  our users' requirements, not by the whims of the core team. (See
+  :ref:`governance`.)
+- We serve scientific applications primarily, over “consumer” image editing in
+  the vein of Photoshop or GIMP. This often means prioritizing n-dimensional
+  data support, and rejecting implementations of “flashy” filters that have
+  little scientific value.
+- We value elegant implementations over getting every last ounce of
+  performance. Readable code that is easy to understand, for newcomers and
+  maintainers alike, makes it easier to contribute new code as well as prevent
+  bugs. This means that we will prefer a 20% slowdown if it reduces lines of
+  code two-fold, for example.
+- We require excellent documentation. All functions should have NumPy-style
```

Comment:
This isn't really a value—how can we better phrase it as one?  We care about the usability of our software?  We strive to write code that is maximally useful?

Reply:
I've changed it to "we value education and documentation"...